plan/plans/info_test.go

```
@@ -68,6 +68,13 @@ func mustQuery(c *C, currDB *sql.DB, s string) int {
 	return cnt
 }
 
+func mustFailQuery(c *C, currDB *sql.DB, s string) {
+	rows, _ := currDB.Query(s)
```

Please don't ignore error.